### PR TITLE
[Repo] Remove duplicate content from SDK README

### PR DESCRIPTION
**Pull request by CodeBlanch**
## Changes

Removes the duplicate content from the OpenTelemetry README and links instead to the `customizing-the-sdk` README(s).

### CODE REVIEW

File: docs/trace/customizing-the-sdk/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Customizing OpenTelemetry .NET SDK
+# Customizing OpenTelemetry .NET SDK for Tracing
```
Comment (cijothomas) [inline docs/trace/customizing-the-sdk/README.md:1]: the docs are under docs/trace folder, so this may not be needed.

Comment (CodeBlanch) [reply docs/trace/customizing-the-sdk/README.md:1]: I poked around some have it, some don't. Metrics does. The reason I added it here is so the link reads better in the SDK README and is consistent with Metrics 🤷

File: docs/trace/customizing-the-sdk/README.md [hunk 2]
```diff
@@ -35,9 +35,15 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder().Build();
 
 // Dispose at application shutdown
 tracerProvider.Dispose()
-
 ```
 
+**Note:** The `Sdk.CreateTracerProviderBuilder()` API is available for all
+runtimes but for `ASP.NET Core` and [.NET Generic
```
Comment (cijothomas) [inline docs/trace/customizing-the-sdk/README.md:41]:
```suggestion
runtimes. For `ASP.NET Core` and [.NET Generic
```

**Review by cijothomas — APPROVED** [review]

File: docs/trace/customizing-the-sdk/README.md (hunk 2 above)
Comment (utpilla) [inline docs/trace/customizing-the-sdk/README.md:41]:
```suggestion
runtimes. Additionally, for `ASP.NET Core` and [.NET Generic
```

File: docs/trace/customizing-the-sdk/README.md [hunk 3]
```diff
@@ -127,9 +133,10 @@ refer to corresponding documentation of the instrumentation library to know the
 exact method name.
 
 Follow [this](../extending-the-sdk/README.md#instrumentation-library) document
-to learn about the instrumentation libraries shipped from this repo.
+to learn about the instrumentation libraries shipped from this repo and writing
+custom instrumentation.
```
Comment (utpilla) [inline docs/trace/customizing-the-sdk/README.md:137]:
```suggestion
custom instrumentation libraries.
```

File: docs/trace/customizing-the-sdk/README.md [hunk 4]
```diff
@@ -195,8 +202,10 @@ exporting purposes, the SDK provides the following built-in processors:
   : This is an exporting processor which passes telemetry to the configured
   exporter without any batching.
 
-Follow [this](../extending-the-sdk/README.md#processor) document
-to learn about how to write own processors.
+Follow [this](../extending-the-sdk/README.md#processor) document to learn about
+writing custom processors. Follow
+[this](../extending-the-sdk/README.md#exporter) document to learn about writing
```
Comment (utpilla) [inline docs/trace/customizing-the-sdk/README.md:207]: We should probably include a blurb for exporters as well under `Processors & Exporters` section.

Comment (cijothomas) [reply docs/trace/customizing-the-sdk/README.md:207]: say something like Exporters are not something TracerProvider is aware of...it only calls the processors, and it is the processor who invoke exporter.

File: src/OpenTelemetry/README.md [hunk 5]
```diff
@@ -57,264 +50,27 @@ which provides the ability to enrich logs emitted with `ILogger` with
...
-It is also possible to configure the `Resource` by using following
-environmental variables:
-
-| Environment variable       | Description                                        |
-| -------------------------- | -------------------------------------------------- |
-| `OTEL_RESOURCE_ATTRIBUTES` | Key-value pairs to be used as resource attributes. See the [Resource SDK specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.5.0/specification/resource/sdk.md#specifying-resource-information-via-an-environment-variable) for more details. |
-| `OTEL_SERVICE_NAME`        | Sets the value of the `service.name` resource attribute. If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence. |
-
-### Sampler
+minutes](../../docs/metrics/getting-started/README.md) to get up and running.
```
Comment (utpilla) [inline src/OpenTelemetry/README.md:58]:
```suggestion
minutes](../../docs/metrics/getting-started/README.md) to get metrics up and running.
```

File: src/OpenTelemetry/README.md [hunk 6]
```diff
@@ -57,264 +50,27 @@ which provides the ability to enrich logs emitted with `ILogger` with
...
-containing the Baggage Propagator and TraceContext propagator. This default
-propagator can be overridden with the below snippet.
+## Getting started with Tracing
 
-```csharp
-using OpenTelemetry;
+If you are new to
+[traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md),
+it is recommended to follow [get started in 5
+minutes](../../docs/trace/getting-started/README.md) to get up and running.
```
Comment (utpilla) [inline src/OpenTelemetry/README.md:69]:
```suggestion
minutes](../../docs/trace/getting-started/README.md) to get traces up and running.
```

File: src/OpenTelemetry/README.md [hunk 7]
```diff
@@ -57,264 +50,27 @@ which provides the ability to enrich logs emitted with `ILogger` with
...
-propagator](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#composite-propagator)
-containing the Baggage Propagator and TraceContext propagator. This default
-propagator can be overridden with the below snippet.
+## Getting started with Tracing
 
-```csharp
-using OpenTelemetry;
+If you are new to
+[traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md),
+it is recommended to follow [get started in 5
```
Comment (utpilla) [inline src/OpenTelemetry/README.md:68]:
```suggestion
it is recommended to follow [getting started in 5
```

File: docs/trace/customizing-the-sdk/README.md (hunk 4 above)
Comment (CodeBlanch) [reply docs/trace/customizing-the-sdk/README.md:207]: I will do this as a follow-up!

File: src/OpenTelemetry/README.md (hunk 7 above)
Comment (CodeBlanch) [reply src/OpenTelemetry/README.md]: Updated

File: src/OpenTelemetry/README.md [hunk 8]
```diff
@@ -57,264 +50,27 @@ which provides the ability to enrich logs emitted with `ILogger` with
 `ILogger` based API will become the OpenTelemetry .NET implementation of
 OpenTelemetry logging.
 
-## Getting started with Tracing
+## Getting started with Metrics
 
 If you are new to
-[traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md),
+[metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md),
 it is recommended to follow [get started in 5
```
[the remark this answers is not in the document]
Comment (CodeBlanch) [reply src/OpenTelemetry/README.md]: Updated

File: src/OpenTelemetry/README.md (hunk 5 above)
Comment (CodeBlanch) [reply src/OpenTelemetry/README.md]: It says "metrics" at the begging of the sentence so it felt a bit redundant to say it at the end as well. I tweaked the whole thing a bit. LMK how you feel about it now.

File: docs/trace/customizing-the-sdk/README.md (hunk 3 above)
Comment (CodeBlanch) [reply docs/trace/customizing-the-sdk/README.md]: Updated

File: src/OpenTelemetry/README.md (hunk 5 above)
Comment (utpilla) [reply src/OpenTelemetry/README.md]: Looks good!

**Pull request merged**